### PR TITLE
Fix pixel spacing #2

### PR DESCRIPTION
**Pull request by SimonBiggs**
This is a follow-on PR to https://github.com/pymedphys/pymedphys/pull/1571

### CODE REVIEW

File: lib/pymedphys/tests/dicom/structure/test_masking.py
Comment (SimonBiggs) [inline lib/pymedphys/tests/dicom/structure/test_masking.py:130]: @sjswerdloff here was the issue within that test

File: lib/pymedphys/tests/dicom/test_coords.py
Comment (SimonBiggs) [inline lib/pymedphys/tests/dicom/test_coords.py:163]: @cedricbelanger Yup, this is perfect!

Thank you so much for all of this work. It is both beautiful, and exceptionally helpful!

**Comment by SimonBiggs** [comment]
Hi All,

The only changes I made on top of @cedricbelanger's was to run black and the other pre-commit tools, as well as fix a separate but related failing test, and lastly, run @cedricbelanger's test locally so that the hash for the download file was created. 

This is an awesome find @cedricbelanger, and thank you so much for jumping in and not only diagnosing the issue, but then providing the fix, as well as writing the automated test for it so that future developers/maintainers don't make the same mistake.

I'll block this merge until both @sjswerdloff and @cedricbelanger approve it. @sjswerdloff for the items he flagged in the parent PR of this PR, and @cedricbelanger to make sure he's happy I didn't butcher his work :slightly_smiling_face:.

**Comment by cedricbelanger** [comment]
@SimonBiggs This is perfect. It is OK on my side :) 

Cédric

**Review by cedricbelanger — APPROVED** [review]

**Review by sjswerdloff — APPROVED** [review]

**Comment by SimonBiggs** [comment]
Awesome :) Thanks @sjswerdloff

**Pull request merged**
